src/blocks/file-blocks/explain/index.tsx

```diff
@@ -101,12 +101,14 @@ function BlockInner(props: FileBlockProps) {
   }, [content, context, start, end]);
 
   return (
-    <div className="relative">
+    <div className={tw(`relative`)}>
       {Object.entries(explanations).map(([line, explanation]) => {
         return (
           <div
             key={line}
-            className="px-3 overflow-y-auto break-words absolute bg-white w-[260px] right-0 z-10"
+            className={tw(
+              `px-3 overflow-y-auto break-words absolute bg-white w-[260px] right-0 z-10`
```

Comment:
this px-3 might be a Primer style

Reply:
Ditto -- wanna make sure I understand whether you mean you want to use the values from the Primer spacing scale or it's legit not a TW class